pages/1.12/release-notes/1.12.3/index.md

```diff
@@ -129,7 +131,10 @@ DC/OS 1.12 includes many new features and capabilities. The key features and enh
 - Officially recommended as a Mesosphere supported installation method with best practices built-in (i.e sequential masters & parallel agents in upgrade).
 - Restructured [Mesosphere installation documentation](https://docs.mesosphere.com/1.12/installing/evaluation/) to organize Mesosphere supported installation methods and Community supported installation methods.
 - Expanded DC/OS upgrade paths enable Mesosphere to skip specific [upgrade paths](https://docs.mesosphere.com/1.12/installing/production/upgrading/#supported-upgrade-paths) within a supported patch version of DC/OS (i.e upgrade from 1.11.1 => 1.11.5 in one move) and to skip upgrade paths between supported major to major versions of DC/OS (for example, enabling you to upgrade from 1.11.7 to 1.12.1 in one move).
-- If you have installed the optional DC/OS Storage Service package, then upgrading from 1.12.0 to 1.12.1 requires you to first follow the storage upgrade instructions provided in [Manually upgrade the DSS package to 0.5.x from 0.4.x](/services/beta-storage/0.5.2-beta/upgrades/). You must upgrade DC/OS storage **before** you upgrade cluster nodes to 1.12.1 to prevent Mesos agents from crashing after the upgrade.
+- If you have installed the optional DC/OS Storage Service package, then upgrading from 1.12.0 to 1.12.1 requires you to first follow the storage upgrade instructions provided in [Manually upgrade the DSS package to 0.5.x from 0.4.x](/services/beta-storage/0.5.2-beta/upgrades/). 
```

Comment:
@Chun @jdef could you verify the storage part.

Reply:
looks about right. the latest DSS release is 0.5.3 but the docs for the upgrade haven't changed so linking to 0.5.2 is probably fine.

Reply:
@jdef So we no longer requires the user to upgrade DSS **before** upgrading the cluster?

Reply:
This case was different. I think it was because the protos changed in an "incompatible" way: `vendor` was added to `Resource.DiskInfo.Source`, but frameworks that weren't synced up w/ the latest protos would generate ACCEPT calls w/ resources that didn't match what was offered.